src/components/Main/Main.tsx

```diff
@@ -71,9 +71,10 @@ async function runSimulation(
   const caseCounts: EmpiricalData = countryCaseCountData[params.population.cases] || []
   const containment: TimeSeries = intervalsToTimeSeries(params.containment.mitigationIntervals)
 
-  // serializeScenarioToURL(scenarioState, params)
```

Comment:
This is the line I mean! I don't think we can remove `params` since it contains the non-stale data being used.

Reply:
I understand, just commented above. To me it seems we're doing something wrong and should look at the problem from another angle.